src/browser/api_protocol/api_handlers/event_listener.ts

```diff
@@ -0,0 +1,265 @@
+import { Acker, Identity, EventPayload, Listener, Func } from '../../../shapes';
+import { addEventListener as addNoteListener } from '../../api/notifications/subscriptions';
+import { addRemoteSubscription, subscribeToAllRuntimes, RemoteSubscriptionProps } from '../../remote_subscriptions';
+import { Application } from '../../api/application';
+import { Channel } from '../../api/channel';
+import { ExternalApplication } from '../../api/external_application';
+import { Frame } from '../../api/frame';
+import { getWindowByUuidName, isLocalUuid } from '../../core_state';
+import { GlobalHotkey } from '../../api/global_hotkey';
+import { Identity as NoteIdentity } from '../../api/notifications/shapes';
+import { noop } from '../../../common/main';
+import { System } from '../../api/system';
+import { Window } from '../../api/window';
+import * as _ from 'underscore';
+import * as apiProtocolBase from './api_protocol_base';
+import ofEvents from '../../of_events';
+
+type Subscribe = (
```

Comment:
Chose this word because:
1. Lowercased `subscription` is used for objects, e.g.: [here](https://github.com/HadoukenIO/core/blob/f5e0a4017d5370285a007948d97a4e3a91f9e796/src/browser/api_protocol/api_handlers/event_listener.ts#L55)
2. It represents an action and is used to describe [this](https://github.com/HadoukenIO/core/blob/f5e0a4017d5370285a007948d97a4e3a91f9e796/src/browser/api_protocol/api_handlers/event_listener.ts#L222)